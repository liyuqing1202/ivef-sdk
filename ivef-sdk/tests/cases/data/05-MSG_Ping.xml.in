<MSG_IVEF xmlns="urn:http://www.ivef.org/XMLSchema/IVEF/1.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="urn:http://www.ivef.org/XMLSchema/IVEF/1.1
\\projsrvr\isis\develop\ivef-sdk\trunk\doc\ivef1.xsd">
	<Header MsgRefId="1234567890123456789012345678901234567890" Version="String"/>
	<Body>
		<Ping TimeStamp="2001-12-17T15:30:47.000Z"/>
	</Body>
</MSG_IVEF>
