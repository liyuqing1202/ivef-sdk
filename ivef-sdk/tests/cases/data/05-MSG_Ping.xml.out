TestApp ready for input
MSG_IVEF
    Header
        MsgRefId = 1234567890123456789012345678901234567890
        Version = String
    Body
        Ping
            TimeStamp = 2001-12-17T15:30:47.000Z
<MSG_IVEF>
<Header MsgRefId="1234567890123456789012345678901234567890" Version="String"/>
<Body>
<Ping TimeStamp="2001-12-17T15:30:47.000Z"/>
</Body>
</MSG_IVEF>

TestApp shutting down
